cpp/include/cudf/io/parquet.hpp

```diff
@@ -869,6 +890,20 @@ class parquet_writer_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Sets the maximum size in bytes for min and max values in the column index.
```

Comment:
Should we point out that it may not be the real max? Desired max? Maybe call out that it may not be able to truncate it in the max case.

Reply:
Good suggestion.  I'll reword it some.